src/components/filters/checkbox.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
   apply: (filtersToApply) => void
   selected?: (string | number)[]
   facet?: { [key: string]: number }
+  tracking?: (value) => void
```

Comment:
Would it make sense to call it `onSelect` or something like that? In the end, it's a function that you call when a value is selected. It's not really limited to tracking.

Reply:
Yes, it makes sense thanks.